Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs

```diff
@@ -4,22 +4,49 @@
 
 namespace HomagConnect.MaterialManager.Contracts.Material.Base
 {
+    /// <summary>
+    /// Image information.
+    /// </summary>
     public class ImageInformation
     {
-        public Uri? Url { get; set; } 
-
-        public Size? Size { get; set; } 
-
+        /// <summary>
+        /// Gets or sets the url.
+        /// </summary>
+        public Uri? Url { get; set; }
+
+        /// <summary>
+        /// Gets or sets the size.
+        /// </summary>
+        public Size? Size { get; set; }
+
+        /// <summary>
+        /// Gets or sets the type.
+        /// </summary>
         public string? Type { get; set; }
 
+        /// <summary>
+        /// Gets or sets the file name.
+        /// </summary>
         public string? FileName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the content type.
+        /// </summary>
         public string? ContentType { get; set; }
```

Comment:
Shouldn't the content type be an enum?

Reply:
we don´t have an enum for that. Are the values all known?

Reply:
Or maybe I confused it wiht Type?

We have at least these:
![image](https://github.com/HomagGroup/HOMAG-Connect/assets/39241151/8cb88d8f-1bf6-405a-81eb-1c67704a8d03)

and this 
![image](https://github.com/HomagGroup/HOMAG-Connect/assets/39241151/6f8407c2-3b2b-4f34-95fc-29a031420bec)

I think now is the chance if we want to have them as enums for the API. I think this makes checks and switches easier and more reliable.